notebooks/02.08-Sorting.ipynb

```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Up to this point we have ever  been concerned mainly with tools to access and operate on array data with NumPy. &nbsp;This section covers algorithms related to sorting values in NumPy arrays. These algorithms are a favorite topic in introductory computer science courses: if you've ever taken one, you probably have had dreams (or, depending on your temperament, nightmares) about&nbsp;<em>insertion sorts</em>,&nbsp;<em>selection sorts</em>,&nbsp;<em>merge sorts</em>,&nbsp;<em>quick sorts</em>,&nbsp;<em>bubble sorts</em>, and many, many more. All are means of accomplishing a similar task: sorting the values in a list or array.</p><p>For example, a simple&nbsp;<em>selection sort</em>&nbsp;repeatedly finds the minimum value from a list, and makes swaps until the list is sorted. We can code this in just a few lines of Python:</p>
<br/>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>